### ISSUE TITLE
Disabling 'Qt' in `ConvexHull`

### ISSUE THREAD

**Issue by stla** [scipy/scipy#15576]
### Is your feature request related to a problem? Please describe.

Hello,

  I find sad that the `Qt` option (triangulation) is always enabled in `ConvexHull`. I did a R package, **cxhull** which wraps **Qhull** too and with which you have the choice to triangulate or not. The non-triangulation option allows me to do something [like that](https://github.com/stla/cxhull#a-four-dimensional-example) or [like that](https://gist.githubusercontent.com/stla/83925fce019921ec12440ad05fdcd213/raw/839458ffc08d4d07ff6c4775c405fa3683866738/RTT.gif).

### Describe the solution you'd like.

Allows to disable `Qt`.

### Describe alternatives you've considered.

Using R.

**Comment by tylerjereddy** [comment]
So the current signature is:

`scipy.spatial.ConvexHull(points, incremental=False, qhull_options=None)`

with the docs indicating `Option “Qt” is always enabled.`. What are you proposing? An additional keyword argument like `qhull_qt=True` that defaults to `True` but can be disabled?

May not be the most elegant design to have two separate keyword arguments for qhull option control, but maintaining backward compatibility is also a responsibility we have.

**Comment by stla** [comment]
Yes, if you want to maintain backward compatibility, I don't see any other solution. But maybe `triangulate=True` is more elegant, and has the advantage to not need to know the names of the Qhull options.

**Comment by tupui** [comment]
We can also better explain the codes in the doc. We can have a central page for that.
